Chat/ClientApp/src/components/FeedbackButton.tsx

```diff
@@ -22,13 +24,19 @@ export const FeedbackButton = (): JSX.Element => {
 
     return (
         <>
+          <TooltipHost
```

Comment:
In the calling app if we have the space we will use the Report a Bug text.

https://github.com/Azure-Samples/communication-services-web-calling-hero/blob/public-preview/Calling/ClientApp/src/components/Header.tsx

![image](https://user-images.githubusercontent.com/79329532/124514472-f5470200-dd91-11eb-9bbd-e03d711d955e.png)